index.bs

```diff
@@ -399,10 +399,10 @@ This method takes the following parameters:
 
 When this method is invoked, the user agent MUST execute the following algorithm:
 
-1. If the {{ScopedCredentialOptions/timeoutSeconds}} member of {{options}} is <a>present</a>, check if its value lies within
-    a reasonable range as defined by the platform and if not, correct it to the closest value lying within that range. Set
-    |adjustedTimeout| to this adjusted value. If {{ScopedCredentialOptions/timeoutSeconds}} was not specified, then set
-    |adjustedTimeout| to a platform-specific default.
+1. If {{ScopedCredentialOptions/timeout}} was specified, check if its value lies within a reasonable range as defined by
```

Comment:
Any reason for deleting the "member of" language? The text you have just shows up as "If timeout was specified" and we received feedback that this is confusing to readers because timeout is not a parameter to makeCredential directly. The "member of" language makes this clearer.